vaxrank/epitope_prediction.py

```diff
@@ -84,15 +86,15 @@ def fm_index_path(genome):
     return os.path.join(cache_dir, '%s_%d_%d.fm' % (
         genome.species.latin_name, genome.release, 2 if six.PY2 else 3))
 
+
 def index_contains_kmer(fm, kmer):
     """
     Checks FM index for kmer, returns true if found.
     """
-    found = False
     for _ in fm.search(kmer):
-        found = True
-        break
-    return found
+        return True
```

Comment:
can just say `return fm.search(kmer)` and get rid of this function, it's only used once

Reply:
I think it returns an iterator over matches, not sure how the boolean conversion logic interacts with that.